### PR TITLE
move shr_file_getLogUnit to shr_log_getLogUnit

### PR DESCRIPTION
**Pull request by jedwards4b**
### Description of changes
Clean up usage of log files

### Specific notes
Depricated shr_file_getLogUnit and added shr_log_getLogUnit
Depends on https://github.com/ESCOMP/CESM_share/pull/36
Contributors other than yourself, if any:

CMEPS Issues Fixed (include github issue #):

Are changes expected to change answers? (specify if bfb, different at roundoff, more substantial) 

Any User Interface Changes (namelist or namelist defaults changes)?

### Testing performed

Testing performed if application target is CESM:
- [ ] (recommended) CIME_DRIVER=nuopc scripts_regression_tests.py
   - machines:
   - details (e.g. failed tests):
- [ ] (recommended) CESM testlist_drv.xml
   - machines and compilers:
   - details (e.g. failed tests):
- [X] (optional) CESM prealpha test
   - machines and compilers cheyenne intel
   - details (e.g. failed tests): all consistant with baselines. 
- [ ] (other) please described in detail
   - machines and compilers
   - details (e.g. failed tests):

Testing performed if application target is UFS-coupled:
- [ ] (recommended) UFS-coupled testing
   - description:
   - details (e.g. failed tests):

Testing performed if application target is UFS-HAFS:
- [ ] (recommended) UFS-HAFS testing
   - description:
   - details (e.g. failed tests):

### Hashes used for testing:

- [ ] CESM:
  - repository to check out: https://github.com/ESCOMP/CESM.git
  - branch/hash:
- [ ] UFS-coupled, then umbrella repostiory to check out and associated hash:
  - repository to check out:
  - branch/hash:
- [ ] UFS-HAFS, then umbrella repostiory to check out and associated hash:
  - repository to check out:
  - branch/hash:

### CODE REVIEW

File: cime_config/namelist_definition_drv.xml [hunk 1]
```diff
@@ -891,7 +891,7 @@
       default: ocn
     </desc>
     <values>
-      <value>ogrid</value>
+      <value>xgrid</value>
```
Comment (billsacks) [inline cime_config/namelist_definition_drv.xml:894]: I don't think this should be part of this PR.

**Review by billsacks — CHANGES_REQUESTED** [review]
Mostly looks good - thanks. But can you please remove the unrelated change of the aoflux grid?

File: cime_config/namelist_definition_drv.xml (hunk 1 above)
Comment (jedwards4b) [reply cime_config/namelist_definition_drv.xml:894]: You are correct - I removed it in my sandbox, but forgot to push that back.

**Review by billsacks — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**